src/shared/lib/transform.js

```diff
@@ -91,7 +91,7 @@ export const transposeTimeseries = function(timeseriesByLocation) {
   return { timeseriesByDate, locations };
 };
 
-const caseFields = ['cases', 'recovered', 'active', 'deaths', 'tested'];
+const caseFields = ['cases', 'recovered', 'active', 'deaths', 'tested', 'hospitalized', 'discharged'];
```

Comment:
Isn't discharged the same as recovered?

Reply:
No, discharged refers to discharged from the hospital. Patients may still be sick and recovering at home. The number of recovered is not available for France.